client/lib/mixins/perf/why-did-you-update/index.js

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable no-console */
```

Comment:
Ah, just noticed you did in the README :) I think we still need a specified licence for it though.